### ISSUE TITLE
Add scalability and scrollbar

### ISSUE THREAD

**Issue by throwaway67341** [helgoboss/helgobox#357]
The popup window for editing a mapping isn't scalable (you can't grab the sides of it as you normally would to scale a window). This wasn't a huge issue before, but in the newest version this window is higher than my screen, rendering the plugin unusable.

**Comment by helgoboss** [comment]
Which OS are you on and which screen resolution do you have?

Scaling is not possible with this kind of window. At a maximum I could try to add scrollbars if the window is larger than the screen.

**Comment by throwaway67341** [comment]
Thanks for the reply!

I am on Windows 10 and my screen is 1920 x 1080 pixels, with the screen scale currently set to 125%.

Scrollbars would be very welcome. I think I worded it wrongly, what I mean by scaling is actually cropping + scrollbars. As in, when I grab an edge of the window and pull it inwards, a scrollbar automatically appears. That's the behavior I'm used to from most windows and would be a perfectly fine fix for my problem.

Ideally the window would also always be at most as large as the screen when it opens up.

**Comment by helgoboss** [comment]
Okay, the 125% scale factor makes the difference. As a workaround you can just press escape for now to leave the dialog, so it stays usable until I come up with a solution. Probably I will implement some show/hide logic rather than introducing a scrollbar.

**Comment by throwaway67341** [comment]
Yeah, I'm working on a laptop, so my screen isn't huge. 125% is actually the recommended setting and I'm even thinking about going to 150%.
By show/hide logic you mean making individual sections of the window in question collapsable? I think that would work for me. I'm eagerly awaiting the fix, thanks!

**Comment by ArthurAttout** [comment]
+1 for a vertical scrollbar on the mapping edition panel. 
It would entirely do the trick (no need to add fancy collapsible sections). 
Currently on Ubuntu with 1366x768 resolution, and I'm missing a few controls at the bottom of the window
![Screenshot from 2023-01-31 20-46-11](https://user-images.githubusercontent.com/18170545/215866570-5b1882dd-0007-4f41-9a51-245dec02ab6f.png)
